frontend/src/views/dev/Controls.vue

```diff
@@ -391,10 +122,115 @@ export default {
     selectValue: null,
     dateValue: '2020-01-01',
     timeValue: '2020-01-01T14:45:00+00:00',
+
+    headers: [
```

Comment:
why are the headers in data?
they don't change, don't they? -> they could be a computed value, which would be cached
but i don't know the best practice here.
https://stackoverflow.com/questions/50536254/vue-js-referencing-constants-in-data

Reply:
This is a config array for the datatable and is best practice. A computed value is only needed if it depends on other props/data and should be recomputed if at least one of the dependencies change. As you can see below for the items array (for instance the `this.textfieldValue` in the `id: 'text-field'` items).